andela/mrm_api
# #160509420 Ratio of Checkins to booked meetings per day

#### What does this PR do?
- Allows a query for checkins ratio to be supplied with a single date and returns the results for that specific day

#### Description of Task to be completed?
- Refactor the bookings ratio function to accept a single value (`startDate`) and return the results for that day

#### How should this be manually tested?
- Run the server `http://127.0.0.1:5000/mrm`
- Add some Checkin data with the same structure as the query in the image below
<img width="1090" alt="screenshot 2018-11-01 at 15 27 49" src="https://user-images.githubusercontent.com/33119403/47851809-ca3ed000-ddea-11e8-8fc9-9ebad22ebf95.png">

- Run the query `analyticsRatios` with a single parameter `startDate` for the same date that you entered in the checkins
- The query should return the Bookings ratio for that day alone

#### What are the relevant pivotal tracker stories?
[#160509420](https://www.pivotaltracker.com/story/show/160509420)

#### Screenshots
**Test with 1 date**
<img width="1105" alt="screenshot 2018-11-01 at 15 21 37" src="https://user-images.githubusercontent.com/33119403/47851552-04f43880-ddea-11e8-945c-87d82fdf2896.png">


**Test with 2 dates**
<img width="1075" alt="screenshot 2018-11-01 at 15 22 46" src="https://user-images.githubusercontent.com/33119403/47851591-27865180-ddea-11e8-99dd-af4485d8842d.png">
